helpers/instructor.helper.js

```diff
@@ -8,5 +9,9 @@ function getInstructor(ucinetid) {
     return cache[ucinetid] ? cache[ucinetid] : null;
 }
 
+function getInstructorFromName(name) {
```

Comment:
We should specify that this returns a ucinetid, not an instructor object. 
```suggestion
function getInstructorUciNetIDFromName(name) {
```

Reply:
ok my suggested function name is highkey pretty ugly LOL. Feel free to come up with a better one!

Reply:
I hope getUCINetIDFromName is good, since only instructors have ucinetids, so it should be implied.